pkg/didcomm/packager/api.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package packager
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/envelope"
```

Comment:
Is there a reason for a separate envelope package? Does it now only have a struct?

Reply:
Envelope struct, Packer interface, and Packer Creator.

Reply:
It's separate mainly to avoid an import cycle - which iirc was a cycle only because of tests, so I'll see if I can fix that.

Reply:
I htink I should keep the packages separate, but I can rename `envelope` to `packer` - I'd named it `packer` earlier. I'll work out how to put the Envelope struct inside `pkg/didcomm/packager` without an import cycle, which I believe only came about due to tests.

Reply:
I've resolved this by pulling `Envelope` out into `pkg/didcomm/common/transport`